circuits/cpp/src/aztec3/circuits/rollup/test_utils/nullifier_tree_testing_harness.cpp

```diff
@@ -3,6 +3,7 @@
 #include <barretenberg/stdlib/merkle_tree/nullifier_tree/nullifier_leaf.hpp>
 #include <cstdint>
 #include <tuple>
+#include "aztec3/circuits/abis/append_only_tree_snapshot.hpp"
```

Comment:
is this needed, seems the function using it is in the hpp file

Reply:
Addressed in https://github.com/AztecProtocol/aztec3-packages/pull/342/commits/0091cd760999ae8ebc760ef640393658ccb36a51